wire-ios-request-strategy/Sources/Request Strategies/Client Message/ClientMessageRequestFactory.swift

```diff
@@ -104,11 +104,12 @@ public final class ClientMessageRequestFactory: NSObject {
         }
 
         let originalPath = "/" + ["conversations", conversationID, "otr", "messages"].joined(separator: "/")
-        guard let encryptedPayload = message.encryptForTransport() else { return nil }
-        let path = originalPath.pathWithMissingClientStrategy(strategy: encryptedPayload.strategy)
-        let request = ZMTransportRequest(path: path, method: .post, binaryData: encryptedPayload.data, type: protobufContentType, contentDisposition: nil, apiVersion: apiVersion.rawValue)
-        request.addContentDebugInformation(message.debugInfo)
-        return request
+        fatalError("replace implementation")
```

Comment:
this will need to be tackle in the other dependent PR

Reply:
Why don't you delete the whole method? AFAIK it's not used any more.